Update gcloudsdk.nuspec

ronca71 opened the pull request:
Added python3 dependency

IvanaGyro commented on gcloudsdk/gcloudsdk.nuspec:
```diff
@@ -6,20 +6,24 @@
 This package always installs the latest version of the Google Cloud SDK from Google. It first retrieves the most current installer, which then fetches the latest version of the SDK and installs it for all users in the default installation path. Updates can be performed through the gcloud cli.
     </description>
+     <dependencies>
++      <dependency id="python3" />
++      <dependency id="chocolatey-core.extension" version="1.1.0" />
++    </dependencies>
```
Here are some redundant '+' signs.
```suggestion
       <dependency id="python3" />
       <dependency id="chocolatey-core.extension" version="1.1.0" />
     </dependencies>
```

IvanaGyro reviewed (changes requested):
The version of 20200527 has not been released before, so we can stay in `0.0.0.20200527` to make the version consistent to the signing time of the executable installer.

I tested this package in [Chocolatey Testing Environment](https://github.com/chocolatey-community/chocolatey-test-environment). It is successfully installed when I tested with the inline provisioner, but I got exit code 2 when I tested with running the command in the box. I had to reopen the shell (or enter `refreshenv`) and to run the install command again to make the installation procees successful. This situation may relate to https://github.com/chocolatey/choco/issues/439. But it seems that the issue was solved years ago. Does we have to concern the user may get exit code 2 if they install this package with the command line without python3 preinstalled?

ronca71 commented:
I suppose "chocolatey-core.extension" dependency is useless. Verified package installation in the Choco Test Env without it and all was completed succesfully.

IvanaGyro commented:
> 
> 
> I suppose "chocolatey-core.extension" dependency is useless. Verified package installation in the Choco Test Env without it and all was completed succesfully.

Yes. chocolatey-core.extension is not necessary. Would you remove it before merging?

ronca71 commented:
- removed choco core extension as dependency
- aligned version to 20200527
- added a comment in chocolateyinstall.ps1 to warn about Python 3.x required, though I expect users should never run PS script manually

IvanaGyro commented on gcloudsdk/gcloudsdk.nuspec:
```diff
@@ -6,20 +6,23 @@
     <id>gcloudsdk</id>
     <!-- Use the signing time of the executable file as the version. -->
-    <version>0.0.0.20200527</version>
+    <version>0.0.0.20205027</version>
```
```suggestion
    <version>0.0.0.202005027</version>
```
Missed a zero.

IvanaGyro replied:
```suggestion
    <version>0.0.0.20200527</version>
```
Here is a typo.

IvanaGyro commented:
@gpoul Could you grant us permission to merge PR?
